tests/meta-mender-toradex-nxp-ci/README

```diff
@@ -0,0 +1,64 @@
+This README file contains information on the contents of the
```

Comment:
Because it will be used for testing only. I don't see any other reasonable method of setting mender storage variables than in machine config. This layer was crated specifically to define `colibri-imx7-mender` machine config and isolate it from `meta-mender-toradex-nxp`.

The `README` is obviously a template added `yocto-layer` which I have not updated yet.